.github/scripts/arm/build.sh

```diff
@@ -4,13 +4,14 @@
 
 [ -z "$1" ] && { echo "Missing 1st argument: PostgREST github commit SHA"; exit 1; }
 [ -z "$2" ] && { echo "Missing 2nd argument: Build environment directory name"; exit 1; }
+[ -z "$3" ] && { echo "Missing 3rd argument: GHC version"; exit 1; }
 
 PGRST_GITHUB_COMMIT="$1"
 SCRIPT_DIR="$2"
 
 DOCKER_BUILD_DIR="$SCRIPT_DIR/docker-env"
-CABAL_VERSION="3.6.0.0"
-GHC_VERSION="9.2.4"
+CABAL_VERSION="recommended"
```

Comment:
This value is weird. Is its interpretation documented somewhere?

Reply:
Yes, according to ghcup, it's the default installation: https://www.haskell.org/ghcup/install/#which-versions-get-installed and it can be used as an installation tag https://www.haskell.org/ghcup/guide/#tags-and-shortcuts. 

Although, now that I check it, the `ubuntu-latest` runner image uses Cabal 3.10.1.0, which is the `latest` version in ghcup. I'll change it to `latest` to keep consistency between both builds.